.github/workflows/flutter.yaml

```diff
@@ -0,0 +1,38 @@
+name: Dart
```

Comment:
The name of this file is not critical; some repos call it `dart.yaml`, some `build.yaml` / `ci.yaml`. Whatever you use, I'd make sure that it agrees with the CI badge at the top of the readme (https://raw.githubusercontent.com/dart-lang/leak_tracker/main/README.md).

Reply:
And fixed the reference. I did not realize it is referenced. That's cool!